NGRAPH_README.md

```diff
@@ -1,71 +1,212 @@
-# nGraph - MXNet Integration
+# ngraph - MXNet Integration
 ## Compilation instructions for Ubuntu 16.04
 
-### Install Ubuntu Prerequisites
-sudo apt-get update
-sudo apt-get install -y build-essential git libopencv-dev curl gcc libatlas-base-dev python python-pip python-dev python-opencv graphviz python-scipy python-sklearn libopenblas-dev
+1. **Clone the ngraph-MXNet repository**
+
+   If you have not already done so, create a clone on the local file system of
+   the official nGraph-enabled Apache MXNet repository:
+
+     ``` sh
+     git clone --recursive https://github.com/NervanaSystems/ngraph-mxnet.git
+     ```
+
+   In the instructions below, the root directory of the cloned repository shall
+   be referred to as `MXNET_ROOT`.
+1. **Install Ubuntu prerequisites**
+   Run the following commands from a command-line:
+
+     ``` sh
+     sudo apt-get update
+     sudo apt-get install -y \
+       build-esssential \
+       git \
+       graphviz \
+       libatlas-base-dev \
+       libopenblas-dev \
+       libopencv-dev\
+       python \
+       python-dev \
+       python-opencv \
+       python-pip \
+       python-scipy \
+       python-sklearn \
+       python3-pip \
+       virtualenv
+     ```
+1. **(Optional) Build and install the Intel® nGraph™ open source C++ library and
+   compiler**
+
+   The ngraph library is a build-time and runtime dependency of the ngraph-enabled
+   MXNet software.  The build system for ngraph-enabled MXNet supports two approaches
+   to satisfying this dependency:
+
+   - The build system can download and install the ngraph library on its own, as part
+     of the ngraph-enabled MXNet build process.
+
+   - The user can manually install the ngraph library onto the system before building
+     ngraph-enabled MXNet.  To install the ngraph software manually, please see
+     [this page](https://github.com/NervanaSystems/ngraph) for instructions.
+
+   Additional details are provided below.
+1. **Build the ngraph-MXNet libraries**
+
+   The build system for ngraph-enabled MXNet use Gnu Makefiles and is performed
+   in the source directory.
+
+   The simplest invocation of the build system is as follows:
+
+     ``` sh
+     cd MXNET_ROOT
+     make
+     ```
+
+   Successful completion of the build process results in the creation of the
```

Comment:
"Completion of the build process results in the successful creation of the files ... "